analysis/db/us_map/choroplethMap.py

```diff
@@ -34,7 +34,10 @@
 molten_df['date_iso'] = pd.to_datetime(molten_df['date'], format="%m/%d/%y")  # change date to ISO8601 standard format
 
 fips = molten_df['fips_str'].tolist()
```

Comment:
Because of the below changes, you don't need this line anymore since you're passing in the column of values into the plotting function